wiki/UI-component/Video.md

```diff
@@ -31,14 +31,75 @@ The [[Video]] component utilizes the embeddable Kaltura player <sup>[[r2](#resou
 <details open="true">
   <summary><strong>Resources</strong></summary><br />
 
-- r1: [Box folder](https://ibm.ent.box.com/folder/108837570522)
+- r1: [Box folder](https://ibm.box.com/s/9q7rdj73m32hufcw1xwl1td6nr88rghm)
 - r2: [Kaltura player](http://player.kaltura.com/docs/api)
 
 <br />
 </details>
 
 <hr width="40%" />
 
-<br />
+## Functions
+
+| Variants | Interaction | Functions |
+| -------- | ----------- | --------- |
+| Default | Clicking on `Poster image` | Poster image is replaced with Kaltura video player and video will begin playing. |  
+| with lightbox media viewer | Clicking on `Poster image` | Open LightBoxMediaViewer and video will begin playing. |
+
+
+## Logics
+
+### Default variant
```

Comment:
Why do we have this level of headings? I don't know if it's necessary, and it doesn't really follow how we're doing other components.